packages/subgraph/schema.graphql

```diff
@@ -1,13 +1,297 @@
-type Token @entity {
+type TenderizeGlobal @entity { # Global Values/ TODO: better name?
+  "Fixed ID of '1'"
   id: ID!
-  decimals: Int
-  name: String
-  symbol: String
+  "List of congifs for all integrations"
+  configs: [ProtocolConfig!]!
+  # TODO: Add global totals 
 }
 
-type Transfer @entity {
+type ProtocolConfig @entity {
```

Comment:
Why distinguish between `ProtocolConfig` and `Protocol` ? 

You can simply use a single entity , preferably the `Protocol` one which uses derived fields using the addresses of the underlying contracts